v3/docs/api/c/meson.build

```diff
@@ -40,6 +36,6 @@ generated_docs += custom_target(
         meson.current_source_dir() / '../.doxybook/templates'
     ],
     depends: [
-        c_doxygen,
+        c_docs,
```

Comment:
The input option points to subprojects/hse-r2.1.1? Is that something we are waiting to tag the r3.0.0?

Reply:
Whoops. No, I am just an idiot! Great catch